lib/trento/application/usecases/clusters/clusters.ex

```diff
@@ -91,4 +112,14 @@ defmodule Trento.Clusters do
     results ->
       {:ok, results}
   end
+
+  @spec update_cib_last_written(String.t(), String.t()) :: {:ok, Ecto.Schema.t()} | {:error, any}
+  def update_cib_last_written(cluster_id, cib_last_written) do
+    case Repo.get(EnrichedCluster, cluster_id) do
+      nil -> %EnrichedCluster{cluster_id: cluster_id}
+      cluster -> cluster
```

Comment:
for consistency `cluster` should be the actual little data-lake we'd use for enrichment.
In this specific case could be `enriched_cluster` or something reflecting a different name if we're going to change it.